tests/flow/test_create_clause.py

```
@@ -113,3 +113,52 @@ def test_03_edge_attributes(self):
+
+        q = """CREATE p=(a)-[:R]->(b), q=(c)-[:S]->(d)
+               RETURN length(p), length(q), size(nodes(q))"""
+        res = self.g.query(q)
+        self.env.assertEquals(res.nodes_created, 4)
+        self.env.assertEquals(res.relationships_created, 2)
+        self.env.assertEqual(res.result_set, [[1, 1, 2]])
+
+        try:
+            self.g.query("CREATE (n) RETURN p")
+            self.env.assertTrue(False)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)